SignalMessaging/contacts/OWSSyncManager.m

```diff
@@ -317,7 +317,10 @@ - (dispatch_queue_t)serialQueue
     static dispatch_queue_t _serialQueue;
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
-        _serialQueue = dispatch_queue_create("org.whispersystems.contacts.syncing", DISPATCH_QUEUE_SERIAL);
+        NS_VALID_UNTIL_END_OF_SCOPE NSString *label = [OWSDispatch createLabel:@"contacts.syncing"];
+        const char *cStringLabel = [label cStringUsingEncoding:NSUTF8StringEncoding];
+
+        _serialQueue = dispatch_queue_create(cStringLabel, DISPATCH_QUEUE_SERIAL_WITH_AUTORELEASE_POOL);
```

Comment:
Bonus change: `DISPATCH_QUEUE_SERIAL` -> `DISPATCH_QUEUE_SERIAL_WITH_AUTORELEASE_POOL`

This is usually wise idea anyway. The dispatch docs seem to imply that objects autoreleased on this queue won't have a pool.
```
 * @const DISPATCH_AUTORELEASE_FREQUENCY_INHERIT
 * Dispatch queues with this autorelease frequency inherit the behavior from
 * their target queue. This is the default behavior for manually created queues.
 ...
 * @const DISPATCH_AUTORELEASE_FREQUENCY_NEVER
 * Dispatch queues with this autorelease frequency never set up an individual
 * autorelease pool around the execution of a block that is submitted to it
 * asynchronously. This is the behavior of the global concurrent queues.
```

From man dispatch_queue_create:
```
The default target queue of all dispatch objects created by the application
     is the default priority global concurrent queue
```